sdk/eventhub/event-hubs/README.md

```diff
@@ -272,7 +274,7 @@ consuming events. Pass the id of the partition you want to target to the `subscr
 
 In the below example, we are using the first partition.
 
-The [subscribe](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.6/classes/eventhubconsumerclient.html#subscribe) 
+The [subscribe](https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.0-preview.7/classes/eventhubconsumerclient.html#subscribe) 
```

Comment:
Lets remove the link to `subscribe` here for the same reason as https://github.com/Azure/azure-sdk-for-js/pull/6353/files#r351551937